lib/fluent/plugin_helper/http_server.rb

```diff
@@ -15,7 +15,6 @@
 #
 
 begin
-  # raise if RUBY_VERSION < 2.3.x. see Gemfile
```

Comment:
`rescue` expression is also not necessary ( also Fluent::PluginHelper::HttpServer::Compat::Server is )